Python/Product/PythonTools/Properties/AssemblyInfo.cs

```diff
@@ -29,20 +29,9 @@
 [assembly: CLSCompliant(false)]
 [assembly: NeutralResourcesLanguage("en-US")]
 
-[assembly: ProvideCodeBase(AssemblyName = "Microsoft.PythonTools", CodeBase = "Microsoft.PythonTools.dll", Version = AssemblyVersionInfo.StableVersion)]
```

Comment:
Are the ProvideCodeBase attributes being moved somewhere else or did you determine we no longer need them at all?  It's making me a bit nervous, don't know what this is going to break if they go away.